### DISCUSSION TITLE
Community meetup recordings availability

### DISCUSSION THREAD

**Discussion by vtslab** [JanusGraph/janusgraph#3026] [Q&A] [answered]
The links to the JanusGraph Online Meetups (hosted at Expero) at https://janusgraph.org/ do not work anymore. Also, when you search for them at the Expero site, you hit a marketing registriation barrier. This is unfortunate, so I guess the Expero PMC members can correct this?

**Comment by porunov** [comment]
@twilmes would you be able to look at it please?

**Reply by twilmes** [reply, marked as answer]
Thanks for the heads up @vtslab @porunov. I'm working on getting the updated links without registration and then I'll update them.
